### DISCUSSION TITLE
The recipient does not exist but the the recipient exist

### DISCUSSION THREAD

**Discussion by vincenzopalazzo** [tutao/tutanota#6443] [Bugs]
- [X] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [X] I've searched and did not find a similar issue.

**Bug in web app**

**Describe the bug**
I receive an email from a guy, but when I try to answer I will get an error from tuta The recipient does not exist.

**To Reproduce**

I should share the email of the guy? there is another way to do this step without posting the email?

**Expected behavior**

It should work as all the other email applications that I am using, e.g: Proton Mail can deliverer the email

**Comment by rezbyte** [comment]
Hi @vincenzopalazzo ,
Sorry to hear you are having issues with replying to an email.

Please report the error by pressing the 'Report' button on the error pop up if you have not already.
If you have a paid plan, please email customer support by using the support dialog inside the Tuta Mail client.

To use the support dialog, press 'Show Help' button (![SupportIcon](https://github.com/tutao/tutanota/assets/48037381/e54cb186-8512-4116-9ef3-7a0d3176f23e)) on the toolbar on the bottom left corner of the screen, then select 'Support'. The support dialog should appear and then you must write your problem into the text box. After some articles appear, scroll to the very bottom of the list & click the button labelled 'Contact support'.
